aws-frauddetector-eventtype/src/aws_frauddetector_eventtype/helpers/api_helpers.py

```diff
@@ -263,16 +315,23 @@ def call_get_variables(frauddetector_client, variable_name: str = None):
     :return: get a single variable if variable_name is specified, otherwise get all variables
     """
     args = {"name": variable_name}
-    validation_helpers.remove_none_arguments(args)
+    args = validation_helpers.remove_none_arguments(args)
     return frauddetector_client.get_variables(**args)
 
 
+@batch_call_limit(request_attribute_to_split="names", response_items_to_collect=["errors", "variables"])
+@api_call_with_debug_logs
+def call_batch_get_variable(frauddetector_client, names: List[str]):
+    args = {"names": names}
+    return frauddetector_client.batch_get_variable(**args)
```

Comment:
BatchGetVariable to get 100 variables at once. Drastically reduces required number of AFD API calls for event types with lots of event variables.